resources/assets.py

```diff
@@ -2388,6 +2393,43 @@ def door_blockstate(base: str) -> JsonObject:
         'facing=west,half=upper,hinge=right,open=true': {'model': top_right_open, 'y': 90}
     }
 
+def log_pile_blockstate() -> JsonObject:
```

Comment:
Why did this need to be permuted like this?

Reply:
This is still just a prototype, I just put it in the first way I thought of since the end goal is to have dynamic models